admin_manual/issues/general_troubleshooting.rst

```diff
@@ -260,14 +260,24 @@ document root of your Web server and add the following lines::
 
     <IfModule mod_rewrite.c>
       RewriteEngine on
-      RewriteRule ^/\.well-known/carddav /nextcloud/remote.php/dav [R=301,L]
-      RewriteRule ^/\.well-known/caldav /nextcloud/remote.php/dav [R=301,L]
-      RewriteRule ^/\.well-known/webfinger /nextcloud/index.php/.well-known/webfinger [R=301,L]
-      RewriteRule ^/\.well-known/nodeinfo /nextcloud/index.php/.well-known/nodeinfo [R=301,L]
+      RewriteRule ^\.well-known/carddav /nextcloud/remote.php/dav [R=301,L]
+      RewriteRule ^\.well-known/caldav /nextcloud/remote.php/dav [R=301,L]
+      RewriteRule ^\.well-known/webfinger /nextcloud/index.php/.well-known/webfinger [R=301,L]
+      RewriteRule ^\.well-known/nodeinfo /nextcloud/index.php/.well-known/nodeinfo [R=301,L]
     </IfModule>
 
 Make sure to change /nextcloud to the actual subfolder your Nextcloud instance is running in.
 
+.. note:: Note, that if you are putting the above configureation directly to
+   the apache configuration (generaly in ``/etc/apache2/sites-enabled/``),
+   you will need to prepend the first argument of each of the above
+   ``RewriteRule`` option with a forward slash ``/``, for example
+   ``^/\.well-known/carddav``.
+   This is because Apache already normalizes the values for the use in
+   ``.htaccess`` by dropping any number of leading slashes, but it does
+   not for the use in its main site configuration where the forward slash
+   is still present.
```

Comment:
Some typos fixed and wording suggestion:
```suggestion
.. note:: If you put the above directives directly into an Apache
   configuration file (usually within ``/etc/apache2/``)
   instead of ``.htaccess``, you need to prepend the first argument of
   each ``RewriteRule`` option with a forward slash ``/``, for example
   ``^/\.well-known/carddav``.
   This is because Apache normalizes patjs for the use in ``.htaccess``
   files by dropping any number of leading slashes, but it does not
   do so for the use in its main configuration files where hence the
   forward slash is needed.
```
In other words: In `.htaccess` one must always use patterns for paths relative to its location, while in central config files one must use patterns which match the absolute path. Somehow makes sense.